node/service/src/chain_spec/moonbase.rs

```diff
@@ -61,6 +61,8 @@ pub fn development_chain_spec(mnemonic: Option<String>, num_accounts: Option<u32
 				// accounts[0],
 				// Council members: Baltathar, Charleth and Dorothy
 				vec![accounts[1], accounts[2], accounts[3]],
+				// Tech comitee members: Alith and Baltathar
+				vec![accounts[1], accounts[2]],
```

Comment:
```suggestion
				vec![accounts[0], accounts[1]],
```

Now it will use Alith and Baltithar as your comment says.